react/__tests__/ProductAssemblyOptions.text.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
Is there a typo in the name of this file?

Reply:
yep